input/admin/config.yml

```diff
@@ -292,6 +292,18 @@ collections:
       - {label: name, name: name, widget: string}
       - {label: key, name: key, widget: string}
       - {label: address, name: address, widget: string}
```

Comment:
If we're breaking out "address" can we get rid of the "address" field itself, or do we need to have both?